### PR TITLE
PHP 8.0 support

### LINKED ISSUE #2: PHP 8.0 support
### Feature Request

|    Q        |   A
|------------ | ------
| New Feature | yes

#### Summary

To be prepared for the december release of PHP 8.0, this repository has some additional TODOs to be tested against the new major version.

In order to make this repository compatible, one has to follow these steps:

- [ ] Modify `composer.json` to provide support for PHP 8.0 by adding the constraint `~8.0.0`
- [ ] Modify `composer.json` to drop support for PHP less than 7.3
- [ ] Modify `composer.json` to implement phpunit 9.3 which supports PHP 7.3+
- [ ] Modify `.travis.yml` to ignore platform requirements when installing composer dependencies (simply add `--ignore-platform-reqs` to `COMPOSER_ARGS` env variable)
- [ ] Modify `.travis.yml` to add PHP 8.0 to the matrix (*NOTE*: Do not allow failures as PHP 8.0 has a feature freeze since 2020-08-04!)
- [ ] Modify source code in case there are incompatibilities with PHP 8.0

### PR DESCRIPTION
**Pull request by svycka**
|    Q          |   A
|-------------- | ------
| New Feature   | yes

closes #2

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,38 +12,28 @@ env:
...
-    - php: 7.2
+    - php: 7.4
       env:
         - DEPS=latest
-    - php: 7.3
+    - php: 8.0
       env:
         - DEPS=lowest
-    - php: 7.3
+        - COMPOSER_ARGS="--no-interaction --ignore-platform-reqs"
```
Comment (svycka) [inline .travis.yml]: laminas-servicemanager does not install without --ignore-platform-reqs I can remove or remove when SM adds support for php8?

Comment (Ocramius) [reply .travis.yml]: Probably best if we first release servicemanager 🤔

Comment (svycka) [reply .travis.yml]: that apply for many other packages and alredy merged so dont think we have to wait

**Comment by svycka** [comment]
@froschdesign why in progress? should be in review?

**Comment by froschdesign** [comment]
@svycka 
> @froschdesign why in progress? should be in review?

The project is automated by GitHub and a review has not yet started.

This is the description from the GitHub project for the column "Review in Progress":

> Pending approval by reviewer
Pull requests in this project will automatically move here when a reviewer requests changes, or it no longer meets the minimum number of required approving reviews. Recommended when another column has the Approved by reviewer automation enabled.

https://github.com/orgs/laminas/projects/11

**Pull request merged**